### DISCUSSION TITLE
Sending Hello and LSU packet

### DISCUSSION THREAD

**Discussion by 03psb** [ANLAB-KAIST/KENSv3#238] [Project Q&A 3. Internet Layer (IP/Routing)] [answered]
while working on the implementation of Hello and LSU packet, I thought I should use UDP header and use Hello and LSU header as payload for UDP packet. Then I should know the destination IP and destination port. Since it is multicast, I guess the destination IP should be 224.0.0.5, but how can I figure out the destination port for multicast?

**Comment by Powering111** [comment, marked as answer]
You don't have to use UDP header, just use PWOSPF header right after Ethernet and IP header. (34 bytes offset)
Therefore, you don't need port or something like that.
